_layouts/home.html

```diff
@@ -4,18 +4,21 @@
 <div class="hero">
   <div class="fixed-width">
     <a class="site-title" href="{{ site.baseurl }}/">
-      <img class="logo" src="{{ '/images/logo-text-white.png' | prepend: site.baseurl }}" alt="Student Robotics"/>
+      <img class="logo" src="{{ '/images/logo-text-white.png' | prepend: site.baseurl }}" alt="Student Robotics" />
     </a>
     {% include navigation.html class="site-navigation home-navigation" %}
   </div>
   <div class="hero-container hero-homepage">
     <div class="fixed-width">
       <div class="hero-text">
         <h1>An annual robotics competition for young people aged 16-18</h1>
-        <a class="button button-hero" href="{{ '/about' | prepend: site.baseurl }}">Learn more</a>
+        <a class="button hero-homepage-button" href="{{ '/compete' | prepend: site.baseurl }}">Compete</a>
+        <a class="button hero-homepage-button" href="{{ '/volunteer' | prepend: site.baseurl }}">Volunteer</a>
+        <a class="button hero-homepage-button" href="{{ 'http://link.justgiving.com/v1/charity/donate/charityId/3118662?tipScheme=TipJar2.1&isRecurring=true&amount=20.00&reference=givingcheckout_tj21' }}">Donate</a>
```

Comment:
```suggestion
        <a class="button hero-homepage-button" href="http://link.justgiving.com/v1/charity/donate/charityId/3118662?tipScheme=TipJar2.1&isRecurring=true&amount=20.00&reference=givingcheckout_tj21">Donate</a>
```

Additionally, is this not defined in a variable somewhere?